src/components/Homepage/Changelog.tsx

```diff
@@ -84,8 +83,16 @@ const stripHtml = (html: string | null | undefined): string => {
   if (!html) {
     return '';
   }
-  const doc = parse(html);
-  return doc.textContent || '';
+
+  const parser = new DOMParser();
+  const doc = parser.parseFromString(html, 'text/html');
+
```

Comment:
also the code assumes that doc.body is always defined, but it could be null?

Reply:
```
  if (!html) {
    return '';
  }
```
I think if `doc.body` is nil, then that code will be nil